README.md

```diff
@@ -1,22 +1,68 @@
 # [Backstage](https://backstage.io)
 
-This is your newly scaffolded Backstage App, Good Luck!
-
 To start the app, run:
 
-```sh
-yarn install
-yarn dev
-```
+# Setup For Development
+
+## One-time Setup
+Install nvm : https://github.com/nvm-sh/nvm#installing-and-updating
+
+Get Node 16: `nvm install 16`
+
```

Comment:
Should there be a comment that if you already have node 16 installed, you merely need `nvm use 16`?